policies/rudderc/tests/cases/general/min.xml

```diff
@@ -0,0 +1,33 @@
+<TECHNIQUE name="NTP">
+  <DESCRIPTION/>
+  <USEMETHODREPORTING>true</USEMETHODREPORTING>
+  <MULTIINSTANCE>false</MULTIINSTANCE>
+  <POLICYGENERATION>separated</POLICYGENERATION>
+  <AGENT type="cfengine-community,cfengine-nova">
+    <BUNDLES>
+      <NAME>min</NAME>
+    </BUNDLES>
+    <FILES>
+      <FILE name="technique.cf">
+        <INCLUDED>true</INCLUDED>
+      </FILE>
+    </FILES>
+  </AGENT>
+  <AGENT type="dsc">
+    <BUNDLES>
+      <NAME>Min</NAME>
```

Comment:
Why "Min" and not "min" here?

Reply:
Because that's what we do on Windows if I read the Scala implementation right, no exact idea why